components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/resources/admin-api.yaml

```diff
@@ -4462,6 +4478,20 @@ components:
             - EXCHANGED
             - ORIGINAL
             - BOTH
+
+    KeyManagerEndpoint:
+      title: Key Manager Endpoint.
+      required:
+        - name
+        - value
+      type: object
+      properties:
+        name:
+          type: string
+          example: 'TokenEndpoint'
+        value:
+          type: string
+          example: 'https://localhost:9443/oauth2/token'
```

Comment:
shall we need to add another field to capture whether that field is required or not, as it may vary for different key managers. WDYT.

Reply:
yes this vary from different key managers